### PR TITLE
Issue 5476 - RFE - add memberUid read aci by default

### LINKED ISSUE #5476: Add memberUid to aci by default
Some legacy methods of group lookup use memberUid which by default we don't allow. Because these searches look for member AND memberUid, the lack of MemberUid access causes the search to fail. 

Add an aci for reading this by default.

### PR DESCRIPTION
**Pull request by Firstyear**
Bug Description: Some legacy client issue searches for (|(member=...)(memberUid=...)). Because of how our aci's work, the lack of access to memberUid (even if empty) would block the legitimate search to member=.

Fix Description: Allow memberUid to be searched by default.

fixes: https://github.com/389ds/389-ds-base/issues/5476

Author: William Brown <william@blackhats.net.au>

Review by: ???

### CODE REVIEW

**Review by mreynolds389 — APPROVED** [review]
ack

File: src/lib389/lib389/configurations/__init__.py [hunk 1]
```diff
@@ -11,17 +11,22 @@
...
+    if (version == INSTALL_LATEST_CONFIG and ds_is_newer('2.3.0')):
+        return c002003000
     if (version == INSTALL_LATEST_CONFIG and ds_is_newer('1.4.2')):
         return c001004002
     elif (version == INSTALL_LATEST_CONFIG and ds_is_newer('1.4.0')):
         return c001004000
     elif (version == INSTALL_LATEST_CONFIG):
         return c001003006
+    elif (version == '002003000' and ds_is_newer('2.3.0')):
+        return c001004002
```
Comment (droideck) [inline src/lib389/lib389/configurations/__init__.py]: Shouldn't it be `return c002003000`?

File: src/lib389/lib389/configurations/config_002003000.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
+# --- BEGIN COPYRIGHT BLOCK ---
+# Copyright (C) 2020 Red Hat, Inc.
```
Comment (droideck) [inline src/lib389/lib389/configurations/config_002003000.py]: It was not the best year, lets forget it :D

**Review by droideck — APPROVED** [review]
Besides a couple of minor issues, LGTM!

File: src/lib389/lib389/configurations/__init__.py (hunk 1 above)
Comment (Firstyear) [reply src/lib389/lib389/configurations/__init__.py]: Opps! Good catch, fixed

File: src/lib389/lib389/configurations/config_002003000.py (hunk 2 above)
Comment (Firstyear) [reply src/lib389/lib389/configurations/config_002003000.py]: Also fixed :)

**Review by droideck — APPROVED** [review]

**Comment by Firstyear** [comment]
Thanks!

**Pull request merged**